src/pages/AllCompletedUncompleted/AllCompletedUncompleted.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const AllCompletedUncompleted = () => {
```

Comment:
Call this component something like `TodoList`.

Reply:
Done.